graphql_client_codegen/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "graphql_client_codegen"
+version = "0.4.0"
```

Comment:
Now I'm aligning the version with other crate, what do you do?

Reply:
I am trying to align the crates on the same version in the rest of the project so it's perfect that way.